src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,22 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * This Command class is the parent class for all commands to be executed.
+ */
+public abstract class Command {
```

Comment:
I like the use of the abstract class Command and how inheritance is used for all the commands! 👍